scripts/composer/ScriptHandler.php

```diff
@@ -44,21 +44,23 @@ public static function createRequiredFiles(Event $event) {
     // Prepare the settings file for installation
     if (!$fs->exists($root . '/sites/default/settings.php')) {
       $fs->copy($root . '/sites/default/default.settings.php', $root . '/sites/default/settings.php');
-      $fs->chmod($root . '/sites/default/settings.php', 666);
+      $fs->chmod($root . '/sites/default/settings.php', 0666);
       $event->getIO()->write("Create a sites/default/settings.php file with chmod 666");
```

Comment:
I updated the comment to say "// Create the files directory with chmod 0777". Does that work?